pandas/tests/frame/test_reshape.py

```diff
@@ -855,21 +855,21 @@ def _test_stack_with_multiindex(multiindex):
                              dtype=df.dtypes[0])
         assert_frame_equal(result, expected)
 
-    def test_stack_preserve_categorical_dtype(self):
+    @pytest.mark.parametrize('ordered', [False, True])
```

Comment:
I think ordered is a fixture already

Reply:
Not in conftest.

Reply:
pandas/tests//arrays/categorical/conftest.py

maybe can simply move it to top level and use it

Reply:
Honestly, I would love to create a helper in `utils/testing.py` to generate boolean fixtures like these on the fly.  Not sure if it's possible, but just throwing it out there for thought.